docker-multinode/common.sh

```diff
@@ -47,10 +51,7 @@ kube::multinode::main(){
   CURRENT_PLATFORM=$(kube::helpers::host_platform)
   ARCH=${ARCH:-${CURRENT_PLATFORM##*/}}
 
-  DEFAULT_NET_INTERFACE=$(ip -o -4 route show to default | awk '{print $5}' | head -1)
-  NET_INTERFACE=${NET_INTERFACE:-${DEFAULT_NET_INTERFACE}}
-
-  DEFAULT_IP_ADDRESS=$(ip -o -4 addr list ${NET_INTERFACE} | awk '{print $4}' | cut -d/ -f1 | head -1)
+  DEFAULT_IP_ADDRESS=$(ip -o -4 addr list $(ip -o -4 route show to default | awk '{print $5}' | head -1) | awk '{print $4}' | cut -d/ -f1 | head -1)
```

Comment:
It was by design; if you want another ip address; just change it. The default hasn't changed from the earlier implementation, but I may add the `NET_INTERFACE` back if you want me to.